x/babylon/types/babylon.go

```diff
@@ -1,7 +1,5 @@
 package types
 
-import "encoding/json"
-
 // Config represents the configuration for the Babylon contract
 type BabylonContractConfig struct {
 	Network                       string `json:"network"`
```

Comment:
Do we still need this struct?

Reply:
No, only needed for tests and moved there. Good catch